api/envoy/extensions/filters/udp/udp_proxy/v3/udp_proxy.proto

```diff
@@ -51,4 +66,9 @@ message UdpProxyConfig {
   // forward all packets from upstream hosts to the Envoy process so that Envoy can forward the packets to the downstream.
   // If the platform does not support this option, Envoy will raise a configuration error.
   bool use_original_src_ip = 4;
+
+  // Optional configuration for UDP proxy hash policy. If hash_policy is not set, the hash-based
+  // load balancing algorithms will select a host randomly. Currently the number of hash policies is
+  // limited to 1.
+  repeated HashPolicy hash_policy = 5 [(validate.rules).repeated = {max_items: 1}];
```

Comment:
`hash_policies` to allow for multiple policies in the future to modify the hash.

Reply:
Yes. That is true.